File: sdk/examples/python/basic_usage.py
#!/usr/bin/env python3
"""CERTEUS Python SDK - Basic Usage Example"""

from certeus_sdk import CerteusAPIError, CerteusClient


def basic_usage():
    """Basic SDK usage example."""
    # Initialize client
    client = CerteusClient(
        base_url='https://api.certeus.dev',
        api_key='your-api-key',
        tenant_id='your-tenant-id'
    )
    
    try:
        # Health check
        health_response = client.health()
        print(f"API Status: {health_response.data['status']}")
        
        # Get OpenAPI specification
        spec_response = client.openapi()
        print(f"API Version: {spec_response.data['info']['version']}")
        
    except CerteusAPIError as e:
        print(f"API Error: {e} (Status: {e.status_code})")
    except Exception as e:
        print(f"Unexpected error: {e}")

if __name__ == "__main__":
    basic_usage()

File: sdk/examples/typescript/basic-usage.ts
// CERTEUS TypeScript SDK - Basic Usage Example
import { CerteusClient } from '@certeus/sdk';

async function basicUsage() {
  // Initialize client
  const client = new CerteusClient({
    baseUrl: 'https://api.certeus.dev',
    apiKey: 'your-api-key',
    tenantId: 'your-tenant-id'
  });

  try {
    // Health check
    const health = await client.health();
    console.log('API Status:', health.data.status);

    // Get OpenAPI specification
    const spec = await client.openapi();
    console.log('API Version:', spec.data.info.version);

  } catch (error) {
    console.error('API Error:', error.message);
  }
}

basicUsage();
